### PR TITLE
Add more details to migration guide

### PR DESCRIPTION
**Pull request by marcospereira**
1. How to update play and sbt
2. Changes to cache apis
3. Better order of migration steps

### CODE REVIEW

File: documentation/manual/releases/release26/migration26/CacheMigration26.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Cache APIs Migration
+
+The Cache API was Rewrite to have a synchronous and an asynchronous version. The old APIs will still work but they are now deprecated.
```
Comment (richdougherty) [inline documentation/manual/releases/release26/migration26/CacheMigration26.md]: `was Rewrite` → `has been rewritten`

File: documentation/manual/releases/release26/migration26/Migration26.md [hunk 2]
```diff
@@ -7,6 +7,44 @@ This is a guide for migrating from Play 2.5 to Play 2.6. If you need to migrate
...
+
+```
+sbt.version=0.13.13
+```
+
+### Play JSON moved to separate project
+
+Play JSON has been moved to a separate library hosted at https://github.com/playframework/play-json. Since Play JSON has no depependencies on the rest of Play, the main change is that the `json` value from `PlayImport` will no longer work in your SBT build. Instead, you'll have to specify the library manually:
+
+```
```
Comment (richdougherty) [inline documentation/manual/releases/release26/migration26/Migration26.md]: (Optional) If you like you can use `scala` syntax highlighting for `sbt` file snippets.

File: documentation/manual/releases/release26/migration26/Migration26.md [hunk 3]
```diff
@@ -7,6 +7,44 @@ This is a guide for migrating from Play 2.5 to Play 2.6. If you need to migrate
...
+libraryDependencies += "com.typesafe.play" %% "play-json" % "2.6.0"
+```
+
+Also, Play JSON has a separate versioning scheme, so the version no longer is in sync with the Play version.
+
+### Guice DI support moved to separate module
+
+In Play 2.6, the core Play module no longer includes Guice. You can add it by adding `guice` to your `libraryDependencies`:
```
Comment (richdougherty) [inline documentation/manual/releases/release26/migration26/Migration26.md]: I think most users will *need* to do this, so maybe I suggest making the language a bit stronger. E.g. `You can add it` → `You will need to configure the Guice module`.

Comment (gmethvin) [inline documentation/manual/releases/release26/migration26/Migration26.md]: And maybe something about this should be noted at the top of the migration guide, since this is essential for most apps.

**Comment by marcospereira** [comment]
This is ok to reviewed again.

File: framework/src/play-cache/src/main/java/play/cache/CacheApi.java [hunk 4]
```diff
@@ -7,7 +7,10 @@
 
 /**
  * The Cache API.
+ *
+ * @deprecated Use SyncCacheApi or AsyncCacheApi. Deprecated since 2.6.0.
  */
+@Deprecated
```
Comment (wsargent) [inline framework/src/play-cache/src/main/java/play/cache/CacheApi.java]: Good change, but wouldn't this be part of another PR?

File: documentation/manual/releases/release26/migration26/CacheMigration26.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+# Cache APIs Migration
```
Comment (wsargent) [inline documentation/manual/releases/release26/migration26/CacheMigration26.md]: If you're starting a new page, you should also update the table of contents:

https://github.com/playframework/playframework/blob/master/documentation/manual/releases/release26/migration26/index.toc

File: framework/src/play-cache/src/main/java/play/cache/CacheApi.java (hunk 4 above)
Comment (marcospereira) [inline framework/src/play-cache/src/main/java/play/cache/CacheApi.java]: I was following the boy scout rule here. But I agree with you and will submit another PR. :-)

File: documentation/manual/releases/release26/migration26/CacheMigration26.md (hunk 5 above)
Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/CacheMigration26.md]: Done. I've added other pages that were missing too. :-)

**Comment by marcospereira** [comment]
@wsargent rebased and ready to be reviewed again.
